eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -162,6 +129,10 @@ elseif ($ChangeLogEntries.Contains($Version))
     LogDebug "Updating ReleaseStatus for Version [$Version] to [$($ReleaseStatus)]"
     $ChangeLogEntries[$Version].ReleaseStatus = $ReleaseStatus
     $ChangeLogEntries[$Version].ReleaseTitle = "## $Version $ReleaseStatus"
```

Comment:
FYI: Another place we will need to update in order to fix https://github.com/Azure/azure-sdk-for-java/issues/23908 as we need to be able to control the header level.